daemon/kmd/wallet/driver/ledger.go

```diff
@@ -209,7 +209,21 @@ func (lw *LedgerWallet) SignTransaction(tx transactions.Transaction, pw []byte)
 
 // MultisigSignTransaction implements the Wallet interface.
 func (lw *LedgerWallet) MultisigSignTransaction(tx transactions.Transaction, pk crypto.PublicKey, partial crypto.MultisigSig, pw []byte) (crypto.MultisigSig, error) {
-	return crypto.MultisigSig{}, errNotSupported
+
+	for i := 0; i < len(partial.Subsigs); i++ {
+		subsig := &partial.Subsigs[i]
+		if subsig.Key == pk{
+			sig, err := lw.signTransactionHelper(tx)
+			if err != nil {
+				return partial, err
+			}
+			subsig.Sig = sig
+			return partial, nil
```

Comment:
It would be good to handle the case when `pk` appears multiple times in `partial.Subsigs`.

Reply:
Should it sign multiple times then? Or should that be an error?

Reply:
The current code (in `sqlite.go`) puts the same signature in each slot.  We should not ask the user to approve the transaction on the Ledger multiple times (one per multisig slot).